Psychtoolbox/PsychCal/MeasureDpi.m

```diff
@@ -1,6 +1,16 @@
-function dpi=MeasureDpi(theScreen)
-% dpi=MeasureDpi([theScreen])
-% Helps the user to accurately measure the screen's dots per inch.
+function [dpi, SP]=MeasureDpi(theScreen, CC)
+% dpi=MeasureDpi([theScreen], [CC]);
+% Helps the user to accurately measure the screen's dots per inch. 
+% INPUT:
```

Comment:
Would be good to use the same help text style as many other PTB functions, e.g.,
'theScreen' optional screen parameter.
'CC' If set to 1, reference object is a credit card. If set to 0, some other object.
Also a few more new lines between INPUT and text, and OUTPUT and text wouldn't hurt.
This mostly, so the auto-generated online documentation on psychtoolbox.org will look better.